qemu/tests/bridge_mirror.py

```diff
@@ -77,28 +76,34 @@ def check_tcpdump(output, src_ip, des_ip, ping_count):
                 reply_num += 1
 
         if request_num != ping_count or reply_num != ping_count:
-            logging.debug(
-                "Unexpect request or reply number. "
-                "current request number is: %d, "
-                "current reply number is: %d, "
-                "expected request and reply number is: %d. "
-                % (request_num, reply_num, ping_count))
+            logging.debug("Unexpect request or reply number. "
```

Comment:
s/Unexpect/Unexpected

Reply:
`unexpected request` or `unexpect request`? Do you want to update it or not? Thanks~

Reply:
Will update it.